content/requests/sign_request_create_request.yml

```
@@ -0,0 +1,152 @@
+---
+title: Create a sign request
+
+type: object
+
+description: |-
+  A request to create a sign request object
+
+required:
+  - signers
+  - source_files
+
+properties:
+  is_document_preparation_needed:
+    type: boolean
+    description: >-
+      Indicates if the sender should be taken into the builder flow to
+      prepare the document, or if the document is already prepared.
```

```suggestion
      prepare the document.
```
I think the phrase is a bit confusing because it could mean it is an additional condition for "true" (i.e. if the document is already prepared, returns true).